sql/migrations/relationship-attributes/up.sql

```diff
@@ -43,5 +43,32 @@ CREATE TABLE bookbrainz.relationship_attribute_set__relationship_attribute (
     attribute_id
   )
 );
-	
+
+INSERT INTO bookbrainz.relationship_type ( id, label, description, link_phrase, source_entity_type, target_entity_type, parent_id, child_order, deprecated, reverse_link_phrase ) 
+VALUES 
+( 70, 'Author', 'Indicates the relationship between author and series', 'is part of', 'Author', 'Series', NULL, 0, false, 'contains'),
+( 71, 'Work', 'Indicates the relationship between work and series', 'is part of', 'Work', 'Series', NULL, 0, false, 'contains'),
+( 72, 'Edition', 'Indicates the relationship between edition and series', 'is part of', 'Edition', 'Series', NULL, 0, false, 'contains'),
+( 73, 'EditionGroup', 'Indicates the relationship between edition group and series', 'is part of', 'EditionGroup', 'Series', NULL, 0, false, 'contains'),
+( 74, 'Publisher', 'Indicates the relationship between publisher and series', 'is part of', 'Publisher', 'Series', NULL, 0, false, 'contains');
+
+INSERT INTO bookbrainz.relationship_attribute_type (id, parent, root, child_order, name, description)
+VALUES
+	  (1, NULL, 1, 0, 'position', 'This attribute is used to position the entity in a series.'),
+	  (2, NULL, 1, 0, 'number', 'This attribute indicates the number of a entity in a series.');
```

Comment:
number attribute is used for just display purpose whereas the position attribute is used for positiong the entities in the series.

example of number attribute in MB: https://musicbrainz.org/series/f243e445-74aa-447a-8dc2-75fc0bc5a6a3  https://musicbrainz.org/series/326682f4-cea7-4355-83c3-f85476282f5a

A similar example in BB: https://i.ibb.co/PWW2hMy/Screenshot-from-2021-06-10-23-15-21.png

With help of position attribute we can sort the relationships and render it in a order positioned by the editor.

Reply:
Those are one and the same, at least in those two MusicBrainz examples:
The "number" attribute is used both for displaying (i.e. the "#" column on the page) and also used for  ordering/positioning in the series.
Is both of these cases the series has ordering type set to "automatic", and that just uses a natural sort on the "number" attribute.
I could possibly see how we would need a separate attribute if we have "manual" ordering type however.
Let's both try to find an example of that to make sure this is how we want to do it.

Reply:
OK, I clarified all this for myself and removed the last questions I had in mind:

So in MB, there are a few separate things:
- ordering type: when "manual", enables manual positioning; when "automatic", disables positioning
- a "These relationships have a specific ordering" checkbox shown only when ordering type is "manual", which enables the position attribute; unchecking this removes the position
- when ordering is manual and checkbox is ticked, up/down arrows are shown on the UI and correspond to an "order" numeric attribute
- separately from all that, the "number" attribute can be added (that's what's shown in the "#" column) or removed without affecting the manual order. It will be used for automatic ordering if it is there.

Here's a manually sorted series I modified on test.mb: https://test.musicbrainz.org/edit/79709799
I ticked the "specific ordering" checkbox and reordered the items.
No "number" attribute involved in any of that, but it could also be added without any effect on the ordering.